deploy/olm-catalog/kogito-operator/manifests/kogito-operator.clusterserviceversion.yaml

```diff
@@ -1144,5 +1144,5 @@ spec:
   maturity: alpha
   provider:
     name: Red Hat
-  replaces: kogito-operator.v0.13.0
-  version: 1.0.0-snapshot
+  replaces: kogito-operator.v1.0.0-snapshot
```

Comment:
IMHO it should be `kogito-operator.v0.16.0` here

Reply:
I see there was a problem in the bump-version. gonna correct that
btw, @ricardozanini seems like 0.16.0 is not in olm-catalog on master :S
See https://github.com/kiegroup/kogito-cloud-operator/tree/master/deploy/olm-catalog/kogito-operator

Reply:
It's not because we haven't run the CI. I was on PTO and @spolti did the release. Can we run the CI now?

Reply:
checking the migrate job

Reply:
the migrate job was executed once by @sutaakar before operatorhub PRs have been merged and PR closed as it was too early.
After that the job was never executed again. doing it now on master and will cherry-pick then on this branch

Reply:
https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/KIE/job/kogito/job/tools/job/kogito-operator-copy-manifests-files/13/console

Reply:
will check and cherry-pick that tomorrow and once all merged, I will activate the 0.17.x nightly

Reply:
@radtriste what did I miss on the 0.16 release?

Reply:
nothing particular. we all forgot to migrate 0.16.0 csv files from 0.16.x branch to master after operatorhub prs were merged ;)

Reply:
ah, ok.

Reply:
PR for 0.16.0 manifests on master => https://github.com/kiegroup/kogito-cloud-operator/pull/619